src/nbd.c

```diff
@@ -521,11 +522,16 @@ static void start_read(struct RaucNBDContext *ctx, struct RaucNBDTransfer *xfer)
 		g_error("unexpected error from curl_multi_add_handle in %s", G_STRFUNC);
 }
 
-static struct curl_slist *gstrv_to_slist(const GStrv strv)
+/* Appends Gstrv elements to curl_slist.
+ * If curs_slist does not exist yet (NULL passed), it will be created. */
```

Comment:
This should be explicit about who will own `initial_list`.

Reply:
Will extend doc